### ISSUE TITLE
Alt+1 Alt+2 ... custom shortcuts don't work (for switching tabs)

### ISSUE THREAD

**Issue by abitrolly** [gnome-terminator/terminator#945]
```
➜  ~ cat ~/.config/terminator/config
[global_config]
  window_state = fullscreen
[keybindings]
  split_vert = <Primary><Shift>k
  switch_to_tab_1 = <Alt>1
  switch_to_tab_2 = <Alt>2
  switch_to_tab_3 = <Alt>3
  help = ""
[profiles]
  [[default]]
    font = Monospace 10
    scrollback_infinite = True
    use_system_font = False
```

**Describe the bug**

After setting Alt+1 Alt+2 shortcuts for switching tabs, they don't work. Only default Ctrl+1 Ctrl+2 work.

**To Reproduce**

1. Set Alt+1 Alt+2 for switching tabs
2. Open new tab
3. Press Alt+1 and Alt+2 to see nothing happes

**Expected behavior**

Expected tabs to switch

**Screenshots**

**Desktop (please complete the following information):**
 - OS or Linux Distribution: Fedora
 - Version 40
 - Display Technology: Wayland

**Additional context**

**Comment by kitzberger** [comment]
@abitrolly, for me setting those keybindings in that config file manually didn't work, but as soon as I use the preference dialog and close that, it did work!

**Comment by vssdeo** [comment]
@abitrolly @kitzberger once you update the config file manually. Did you try to relaunch the terminal and check if the bindings work ? When you do via the preferences the bindings are updated that may not be the case with manual update of config file.

**Comment by abitrolly** [comment]
@vssdeo I didn't update config manually. It is just bug report template urges to attach it. I just found that shortcuts actually work, but after changing them, Terminator needs to be restarted. I also haven't found how to disable shortcuts.

**Comment by kitzberger** [comment]
@vssdeo, you're right, after restarting terminator keybindings do work. Can't say whether I restarted the app before.
